frontend/src/features/admin-form/settings/components/AuthSettingsSection/AuthSettingsSingpassSection.tsx

```diff
@@ -23,14 +23,13 @@ export const AuthSettingsSingpassSection = ({
         settings={settings}
         isDisabled={isFormPublic || containsMyInfoFields}
       />
-      {/* Hide the NRIC mask toggle if they have not yet enabled it as part of
-      PMO circular */}
-      {settings.isNricMaskEnabled ? (
```

Comment:
Regarding the rollout of this NRIC Mask, I'm thinking if we could consider retaining this so that we could rollout safely. 

Rough plan:
1. Release nric collection feature with toggle (all forms are defaulted to collect / opt-out)[A]
2. Run datafix `{ set: { isSubmitterIdCollectionEnabled: true }, where: { ... } }` on all form
3. Toggle on feature flag to reveal opt-out


[A] 
```
// form.model.ts
const FormSchema = new Schema<IFormSchema, IFormModel>(
    {
      ...
      isSubmitterIdCollectionEnabled: {
        type: Boolean,
        default: true, // was false
      },
      ...
    }
)
```

Reply:
This can be an exercise to plan for releasing features with zero downtime.

Reply:
Noted! We will deploy according to what we discussed and documented in the PRD!